<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>355</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>290</width>
    <height>355</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>290</width>
    <height>355</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>290</width>
    <height>355</height>
   </size>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Гильфанов 612 3в.</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>331</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="accessibleName">
      <string/>
     </property>
     <attribute name="title">
      <string>Генератор паролей</string>
     </attribute>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>7</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="wrapping">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>4</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_GenPass">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>241</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Сгенерировать пароли</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Введите число символов в пароле:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_SaveFile_Pass">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>241</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Сохранить в файл</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_PassList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>241</width>
        <height>171</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>35</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Спец. символы:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_PassType">
      <property name="geometry">
       <rect>
        <x>106</x>
        <y>30</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Шифр Виженера</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Фраза:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>128</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Ключ:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Key">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>124</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Shifr">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Шифровать</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_DeShifr">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>150</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Дешифровать</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_SaveFile_Shifr">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>241</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Сохранить в файл</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_Text">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>241</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_Shifr">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>241</width>
        <height>81</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_shifrLang">
      <property name="geometry">
       <rect>
        <x>182</x>
        <y>123</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
